Gordon360/ApiControllers/HousingController.cs

```diff
@@ -25,10 +25,71 @@ public HousingController()
             _accountService = new AccountService(_unitOfWork);
         }
 
+        /// <summary>Check if the currently logged in user is authorized to view the housing staff page for applications</summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("staff")]
+        public IHttpActionResult CheckHousingStaff()
+        {
+            //get token data from context, username is the username of current logged in person
+            var authenticatedUser = this.ActionContext.RequestContext.Principal as ClaimsPrincipal;
+            var username = authenticatedUser.Claims.FirstOrDefault(x => x.Type == "user_name").Value;
+
+            //! Placeholder
+            var result = false; // This feature is not yet implemented
+            return Ok(result);
```

Comment:
Should this return NotFound if result is empty?

Reply:
Yes, definitely

Reply:
Wait, that is the endpoint you just made in the other branch (`CheckHousingStaff` --> `CheckIfHousingAdmin`)

Reply:
Oh, right, I fixed it there. I meant to comment on the result in GetApplicationID

Reply:
And in GetUserApplicationID, too?

Reply:
Yes, definitely. All of the HttpGet and HttpPut should use a `NotFound` whenever it fails to find a matching database entry for the given identifier (`username` in the case of `GetUserApplicationID` and `GetApplicationID`, and `ApartAppID` in the case of `EditApplication`)